src/main/java/org/utplsql/cli/LoggerConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package org.utplsql.cli;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.ConsoleAppender;
+import com.zaxxer.hikari.HikariDataSource;
+import org.slf4j.LoggerFactory;
+import org.utplsql.api.TestRunner;
+
+class LoggerConfiguration {
+
+    static void configure(boolean silent, boolean debug) {
```

Comment:
Why do we use two parameters here? Are all 4 combination of their values valid? Maybe we should accept 3-valued enum? 
```
SILENT, DEFAULT, DEBUG
```

Reply:
Reason is to make it easy to just pass the parameters of the RunCommand to the LoggerConfigurator. 
Enum would increase clarity of the LoggerConfigurator class taken alone but not its usage. This way it's clear that logging is depenent on two flags.
If we can increase clarity on the fact that silent is superior to debug, I'd appreciate it.

Reply:
I think we should. If non flags are passed then default logging is enabled, if silent - non, if debug - debug. And then we end up with 3 levels, set on application level and non of other classes should think about it. Classes just log using appropriate log level (`logger.info`, `logger.debug`).

Reply:
I'm not sure if I get you completely right.
You'd like to introduce a 3-leveled ENUM and shift the logic which configuration to take to the RunCommand?
The classes themselves already use appropriate log level, the purpose of the configuration is to either hide or show the log information.